src/main/java/com/minecolonies/api/compatibility/CompatibilityManager.java

```diff
@@ -144,16 +142,6 @@ public class CompatibilityManager implements ICompatibilityManager
      */
     private static ImmutableList<ItemStack> allItems = ImmutableList.of();
 
-    /**
```

Comment:
Thinking about this, maybe we scrap this setting entirely? We already have a tag for the blocks, so the freeblock setting is actually redundant. And a global free pos setting seems kinda useless.

Reply:
Find by me, I can delete it all